lang/java/reef-runtime-azbatch/src/main/java/org/apache/reef/runime/azbatch/client/AzureBatchJobSubmissionHandler.java

```diff
@@ -55,13 +49,14 @@
 
   private static final Logger LOG = Logger.getLogger(AzureBatchJobSubmissionHandler.class.getName());
 
+  private static final String JOB_FOLDER_NAME = "apps/reef/jobs/";
```

Comment:
Take this to ReefFileNames class and refer to that instead.

Reply:
I wasn't sure if they intended ReefFileNames to contain runtime-specific values. But since I do see yarn-specific variables, I'll go ahead and change it.